clarifai/client/dataset.py

```diff
@@ -277,71 +298,104 @@ def _upload_inputs_annotations(
 
   def _retry_uploads(self, failed_input_ids: List[int],
                      retry_annot_protos: List[resources_pb2.Annotation],
-                     dataset_obj: ClarifaiDatasetType) -> None:
+                     dataset_obj: ClarifaiDatasetType, batch_no: Optional[int], **kwargs) -> None:
     """Retry failed uploads.
 
     Args:
       failed_input_ids: failed input ids
       retry_annot_protos: failed annot protos
       dataset_obj: ClarifaiDataset object
+      max_retries: maximum number of retries for failed uploads
     """
+    max_retries = 2
```

Comment:
We can move this to https://github.com/Clarifai/clarifai-python/blob/master/clarifai/constants/dataset.py